### DISCUSSION TITLE
스터디 가입 신청합니다

### DISCUSSION THREAD

**Discussion by Youth787** [ghdcksgml1/Algorithm_Study#275] [스터디 가입 신청]
안녕하세요. 싸피 10기에서 알고리즘을 공부하고 있습니다. 
백준 아이디는 ar77gt입니다.

**Comment by ghdcksgml1** [comment]
안녕하세요! 스터디는 현재 종료되어 자율적으로 학습하시면 될것 같습니다 ㅎㅎ 관심가져주셔서 감사해요😁

네이버 메일앱에서 보냈습니다.
-----Original Message-----
보낸사람: ***@***.***>
받는사람: ***@***.***>
참조: ***@***.***>
날짜: 2023. 8. 2 13:48:12
제목: [ghdcksgml1/Algorithm_Study] 스터디 가입 신청합니다 (Discussion #275)

안녕하세요! 싸피 10기 비전공 자바반에서 알고리즘을 공부하고 있습니다.
백준 아이디는 ar77gt입니다.
